plugins/flytekit-duckdb/flytekitplugins/duckdb/task.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         name: str,
         query: Union[str, List[str]],
         inputs: Optional[Dict[str, Union[StructuredDataset, list]]] = None,
+        md_secret: Optional[Secret] = None,
```

Comment:
If we want to have a motherduck specific parameter, I think it's better to be explicit with the name:

```suggestion
        motherduck_secret: Optional[Secret] = None,
```

`md_secret` looks like "markdown secret"